documentation/types.ts

```diff
@@ -1,35 +1,37 @@
-export interface Snippet {
-  name: string;
+export interface AutocompleteSnippet {
   triggerWords: string[];
   content: string;
   codeFile: string;
 }
 
-export interface CompiledSnippet {
-  name: string;
+export interface CompiledAutocompleteSnippet {
+  // The first item in triggerWords is used as the autocomplete label in the packs IDE
   triggerWords: string[];
   content: string;
   code: string;
 }
 
-export enum Category {
-  ColumnFormat = 'ColumnFormat',
-  Authentication = 'Authentication',
-  DynamicSyncTable = 'DynamicSynctable',
-  Formula = 'Formula',
-  SyncTable = 'SyncTable',
+export interface ExampleSnippet {
+  content: string;
+  codeFile: string;
+}
+
+export interface CompiledExampleSnippet {
+  name: string;
+  content: string;
+  code: string;
 }
 
 export interface Example {
+  tokens: string[];
```

Comment:
Should still be called `triggerTokens` I think? Their use is for triggering the example.

Should also call the autocomplete triggers `triggerTokens` for consistency.